bridges/MozillaSecurityBridge.php

```diff
@@ -21,7 +21,7 @@ public function collectData(){
 			$item['title'] = $element->innertext;
 			$item['timestamp'] = strtotime($element->innertext);
 			$item['content'] = $element->next_sibling()->innertext;
-			$item['uri'] = self::URI;
+			$item['uri'] = self::URI . '?' . $item['timestamp'];
```

Comment:
#1017 recently got merged, so this should work now:

```suggestion
			$item['uri'] = self::URI;
			$item['uid'] = self::URI . '?' . $item['timestamp'];
```

Note: `$item['uid']` should receive a string that uniquely identifies the item (and never changes between calls).